build-system/pr-check/utils.js

```diff
@@ -32,14 +32,19 @@ const {
   travisPullRequestSha,
 } = require('../travis');
 const {execOrDie, exec} = require('../exec');
+const {replaceUrls} = require('../tasks/pr-deploy-bot-utils');
 
 const BUILD_OUTPUT_FILE = isTravisBuild()
   ? `amp_build_${travisBuildNumber()}.zip`
   : '';
 const DIST_OUTPUT_FILE = isTravisBuild()
   ? `amp_dist_${travisBuildNumber()}.zip`
   : '';
-const OUTPUT_DIRS = 'build/ dist/ dist.3p/ EXTENSIONS_CSS_MAP';
+
+const BUILD_OUTPUT_DIRS = 'build/ dist/ dist.3p/ EXTENSIONS_CSS_MAP';
+const DIST_OUTPUT_DIRS =
+  'build/ dist/ dist.3p/ EXTENSIONS_CSS_MAP examples/ test/manual/';
```

Comment:
There's also `dist.tools/`, which contains the experiments page. Maybe include it too?

Reply:
Done. Somewhat related, the zipped dist package increased from 7MB to 65MB with all the HTML files, maybe something to keep an eye on?

Reply:
I see. It would be interesting to know how those files might be used when a version is deployed. (In prod, it serves https://cdn.ampproject.org/experiments.html.)

If it's not likely to be used, dropping the directory SGTM.